src/main/target/common_post.h

```diff
@@ -448,6 +448,29 @@
 #endif
 #endif
 
+// LED strip support
+#if defined(USE_LIMITED_LED_STRIP_OUTPUTS) && defined(USE_LED_STRIP_64)
```

Comment:
Why not just move this to F411 target.h

Reply:
@blckmn , code now follows your suggestion. ready for 3rd reviewer.